Framework/Algorithms/src/ReflectometryReductionOneAuto2.cpp

```diff
@@ -103,6 +113,42 @@ ReflectometryReductionOneAuto2::validateInputs() {
   return results;
 }
 
+/** Validate output workspaces.
+ */
+std::map<std::string, std::string>
+ReflectometryReductionOneAuto2::validateOutputWorkspaces() const {
+  std::map<std::string, std::string> results;
+  bool const isDebug = getProperty("Debug");
+  if (!isDebug && !isChild()) {
+    if (isDefault("OutputWorkspaceBinned")) {
+      results["OutputWorkspaceBinned"] =
+          "OutputWorkspace property must be set when Debug is not enabled.";
```

Comment:
Again, I don't think we need to force the user to enter this because we can use the default name.